x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -196,3 +201,98 @@ export const getColumns = ({
 
   return hasNoPermissions ? cols : [...cols, ...actions];
 };
+
+export const getMonitoringColumns = (): RulesStatusesColumns[] => {
+  const cols: RulesStatusesColumns[] = [
+    {
+      field: 'name',
+      name: i18n.COLUMN_RULE,
+      render: (value: RuleStatus['current_status']['status'], item: RuleStatusRowItemType) => {
+        return (
+          <EuiLink data-test-subj="ruleName" href={getRuleDetailsUrl(item.id)}>
+            {value}
+          </EuiLink>
+        );
+      },
+      truncateText: true,
+      width: '24%',
+    },
+    {
+      field: 'current_status.bulk_create_time_durations',
+      name: i18n.COLUMN_INDEXING_TIMES,
+      render: (value: RuleStatus['current_status']['bulk_create_time_durations']) => (
+        <EuiText data-test-subj="bulk_create_time_durations" size="s">
+          {value?.join(',')}
```

Comment:
I thought about making this an average (mean) but was unsure if just seeing the list of times was better? Thoughts?

Reply:
why not doing a max and change the columns name to reflect that and maybe you can have a tooltip to show all the value.